src/Tribe/Capabilities.php

```diff
@@ -1,38 +1,106 @@
 <?php
 
 class Tribe__Events__Capabilities {
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber',
+	];
+
+	/**
+	 * An array of capabilities aliases by role.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $cap_aliases = [
+		'editor'      => [ // full permissions to a post type
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: dbb401b2b3d14e2c998bacd02d371b92</sub>